UPGRADE.md

```diff
@@ -0,0 +1,114 @@
+If you are not using the disk backed Kubo storage first migrate data to an on disk store following Kubo's instructions.
+
+These IPFS blocks are the raw pieces of Ceramic Events.
+In order to preserve this data we need to migrate them into Ceramic Events stored in as a Sqlite database file.
+
+The `ceramic-one` command contains a subcommand for migrating data from IPFS blocks.
+Install the `ceramic-one` binary following the instructions in README.md.
+
+See the complete usage information for the migration command:
+
+    ceramic-one migrations from-ipfs --help
+
+The command requires an input directory where the IPFS blocks are found and where the output storage directory.
+
+The following command will migrate data assuming the default directories for both Kubo and `ceramic-one`.
+Adjust the command arguments as needed by your infrastructure.
+
+    ceramic-one migrations from-ipfs -i ~/.ipfs/blocks -o ~/.ceramic-one --network mainnet
+
+This command will migrate all blocks stored in the `~/.ipfs/blocks` directory into the database file `~/.ceramic-one/db.sqlite3` for the `mainnet` network.
+As output from the command you should see a total number of events migrated, this will not be the same as the number of blocks as multiple blocks construct into a single event.
+You will also see a number of errors. This represents the number of blocks found that are part of a Ceramic Event but for whatever reasons could not be constructed into a complete event.
+Any error will be logged for investigations.
+Any blocks stored in Kubo that are not Ceramic Events will be ignored.
+
+### Start `ceramic-one` Daemon
+
+Once the data is migrated its time to start the node again.
+The `ceramic-one` process is started in place of the `kubo` process, Kubo is no longer needed.
+
+Read the full usage of the `ceramic-one` daemon:
+
+    ceramic-one daemon --help
+
+Once you are familiar with configuring the daemon start it with defaults or specify any arguments specific to your infrastructure.
+
+    ceramic-one daemon
+
+You should see logs indicating the daemon is running.
+
+### Configure `js-ceramic` for `ceramic-one`
+
+First change the `ipfs` section in the `daemon.config.json` to use `remote` mode and point it at the running `ceramic-one` daemon.
+By default the `ceramic-one` daemon listens on port 5001, if you configured it differently make sure to match that configuration for `js-ceramic` as well.
+
+```josn
+"ipfs": {
+    "mode": "remote",
+    "host": "http://localhost:5001"
+  },
+```
+
+Second, set and environment variable for the `js-ceramic` process so it knows its talking to `ceramic-one` and not Kubo.
+
+    CERAMIC_RECON_MODE=true
+
+Both of these configuration options may not be required in the future, however for now they are.
+
+
+### Start `js-ceramic` Node and Validate
+
+Now start the `js-ceramic` node with its new configuration and environment.
+Validate the process starts as before and that your application can communicate with it as needed.
```

Comment:
Can we include the pointers of what exactly devs should check here? Anything else that would confirm developers that the migration has completed successfully?